packages/react-components/src/atoms/Checkbox.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import css from '@emotion/css';
+import { renderToStaticMarkup } from 'react-dom/server';
+
+import { perRem } from '../pixels';
+import { fern, lead, pine, steel } from '../colors';
+import { noop } from '../utils';
+import { tickIcon } from '../icons';
+
+const checkboxStyles = css({
+  boxSizing: 'border-box',
+  width: `${24 / perRem}em`,
+  height: `${24 / perRem}em`,
+  marginRight: `${12 / perRem}em`,
+  marginTop: `${12 / perRem}em`,
+  marginBottom: `${12 / perRem}em`,
+
+  appearance: 'none',
+  outline: 'none',
+  borderRadius: 0,
+  borderStyle: 'solid',
+  borderWidth: `${1 / perRem}em`,
+  borderColor: steel.rgb,
+
+  ':enabled:hover, :focus': {
```

Comment:
Ive gone back to James' design document and it works as intended as far as i can see

Reply:
I don't doubt that it works as per spec, I'm complaining about the `:focus` here being dead because the `borderColor` on `:focus` is being overwritten further down anyway

Reply:
I would suspect that the intention is
```suggestion
  ':enabled:hover, :enabled:focus': {
```
though?